src/util/helpers/isCompoundVisible.js

```diff
@@ -0,0 +1,14 @@
+/**
+ * Checks whether a compound should be shown based on the visibility of its children
+ *
+ * @param field the compound field
+ * @param data the entire data of the form
+ */
+export default function isCompoundVisible (field, data) {
+  return field.children.some(child => {
```

Comment:
add flow type to child ?

Reply:
Children in a field are created through recursively calling `generateFormSchemaField()` which is already flow typed to return an object of type `FormField`